frontend/src/pages/create_campaign/Roles/Question.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { IconButton, TextField } from "@mui/material";
+import ClearIcon from "@mui/icons-material/Clear";
+import { QuestionTitle, QuestionContent } from "./rolesTab.styled";
+
+const Question = ({
+  questionNumber,
+  question,
+  handleQuestionInput,
+  onQuestionDelete,
+}) => (
+  <>
+    <QuestionTitle>Question {questionNumber}</QuestionTitle>
+    <QuestionContent>
+      <TextField
+        fullWidth
+        multiline
+        required
+        value={question.text}
+        onChange={(e) => {
+          handleQuestionInput(e, question.id);
```

Comment:
can you do `(e) => handleQuestionInput(...)` ?

Reply:
For this and all other cases with the newline in arrow function, eslint complains unless it is formatted this way. I can disable eslint next line or smth if you think it's important - or can change the eslint config if you prefer?